Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -396,3 +396,31 @@
         conditions:
         - !type:ReagentThreshold
           min: 20
+
+- type: reagent
+  id: Coffein
+  name: reagent-name-coffein
+  group: Narcotic
+  desc: reagent-desc-coffein
+  physicalDesc: reagent-physical-desc-strong-smelling
+  flavor: bitter
+  color: "#000000"
+  metabolisms:
+    Poison:
+      effects:
+      - !type:Jitter
+        conditions:
+        - !type:ReagentThreshold
+          min: 3
+      - !type:HealthChange
+        conditions:
+        - !type:ReagentThreshold
+          min: 5
+        damage:
+          types:
+            Poison: 1
+    Narcotic:
+      effects:
+      - !type:MovespeedModifier
+        walkSpeedModifier: 1.01
+        sprintSpeedModifier: 1.02
```

Comment:
Not sure if it's a good idea to have an easy-to-access chem to change your speed. Even if its a tiny amount

Reply:
Does caffeine only have to produce negative effects from an overdose?